subworkflows/local/fastqc.nf

```diff
@@ -8,9 +8,16 @@ workflow FASTQC_CHECK {
   ch_fastq
 
   main:
-  ch_fastq
-      .map { ch -> [ ch[0], ch[1] ] }
-      .set { ch_fastq }
+
+  if( params.aligner == 'cellrangerarc'){
```

Comment:
Would this work? Sincere question since I never tried, but I think would look cleaner if yes.
```groovy
def n = (params.aligner == 'cellrangerarc') ? 3 : 1
ch_fastq.map { ch -> [ ch[0], ch[n] ] }.set { ch_fastq }
```

Reply:
To your first question

> If they are the same, does it make sense to have different input parameters? Is it a flavour choice so that it is possible to split/differentiate them in different sections ( e.g. 'cellrangerarc_options' ) in the schema?

Well we can use the same parameter. I just got an error based on the nextflow_schme.json if I added cellranger_index aslo to cellrangerarc_options because it was already present for cellranger_options.

Reply:
I removed cellrangrarc_index

Reply:
The other snippet works!